src/test/fixtures/profiles/text-to-profile.js

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I ended up not using this code yet. I could rewrite some of the existing tests to use it, and drop some of the other fixture code. Let me know if you want me to remove this from the PR. I have a feeling it will come in handy on the other data transformation requests.